### PR TITLE
switch to PythonBundle & enable use_pip in old dask easyconfig files

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/09dc7d550b092e4cd2da43446861515c for a full test report.

File: easybuild/easyconfigs/d/dask/dask-0.12.0-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -24,11 +23,4 @@ exts_list = [
     }),
 ]
```
Comment (akesandgren) [inline easybuild/easyconfigs/d/dask/dask-0.12.0-intel-2016b-Python-2.7.12.eb]: Exts_list lacks checksums here and some other places

File: easybuild/easyconfigs/d/dask/dask-0.17.0-foss-2017a-Python-2.7.13.eb [hunk 2]
```diff
@@ -31,6 +30,4 @@ sanity_check_paths = {
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/d/dask/dask-0.17.0-foss-2017a-Python-2.7.13.eb]: Sanity check not needed in PythonBundle

Comment (boegel) [reply easybuild/easyconfigs/d/dask/dask-0.17.0-foss-2017a-Python-2.7.13.eb]: fixed in 21c110b5b5

File: easybuild/easyconfigs/d/dask/dask-0.12.0-intel-2016b-Python-2.7.12.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/d/dask/dask-0.12.0-intel-2016b-Python-2.7.12.eb]: fixed in 21c110b5b5

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8a3340b0d72e5032ec3d9c54b5f054a8 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**